pykechain/models/property_reference.py

```diff
@@ -17,9 +17,11 @@ def value(self):
 
     @value.setter
     def value(self, value):
-        if value and not isinstance(value, Part):
-            raise ValueError("Reference must be a Part")
-
-        part_id = value.id if value else None
+        if isinstance(value, Part):
+            part_id = value.id
+        elif isinstance(value, (type(None), str)):
```

Comment:
It seems that this line if not py27 compatible. We get a unicode back from the response, so we need to improve this check.